### ISSUE TITLE
Perform minor dependency updates

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/placement-exams#33]
Prior to the release, we should do a quick check for minor and patch releases for `pypi` dependencies. Here's a breakdown of what could be updated and whether we should.

Package | Now | Latest (as of 2020/08/05) | Decision | Notes
-- | -- | -- | -- | --
`coverage` | `5.1` | `5.2.1` | `5.2.1` | We're only using this as a wrapper over the test runner; I tested the latest release and nothing broke. Seems safe to update to the latest.
`Django` | `3.0.7` | `3.1` | `3.0.9` | `3.1` was released very recently and has no patch releases. I think it's better to just bump to the latest patch of `3.0` and postpone `3.1` to the next update.
`mysqlclient` | `1.4.6` | `2.0.1` | `1.4.6` | The major release took place in July, and there are no new minor or patch releases to pick up. Seems better to wait on this one, maybe update at the same time as the Django `3.1` move.
`python-dotenv` | `0.13.0` | `0.14.0` | `0.14.0` | The usage of this dependency is limited to core functionality, so the minor release should be safe to update. The `v2.0` release of `api-utils-python` also supports this.

**Comment by ssciolla** [comment]
QAed as part of end-to-end tests on 2020-08-06.
